System.IO.Abstractions.TestingHelpers/MockFileStream.cs

```diff
@@ -49,12 +50,14 @@ public MockFileStream(IMockFileDataAccessor mockFileDataAccessor, string path, S
         public override void Close()
         {
             InternalFlush();
+            disposed = true;
         }
 #else
         protected override void Dispose(bool disposing)
         {
             InternalFlush();
```

Comment:
Should we just call `Close`here?

Reply:
Not sure what you're suggesting, calling `Close` from the `Dispose`? The branching seems to be here for some purpose as the double disposing does not crash in NET40, only in netcore.